src/transform/transform.ts

```diff
@@ -42,7 +42,33 @@ async function process(session: Session<CodeModel>) {
 function schemaTypeToGoType(schema: Schema): string {
   switch (schema.type) {
     case SchemaType.Array:
-      return `[]${(<ArraySchema>schema).elementType.language.go!.name}`;
+      const arraySchema = <ArraySchema>schema
+      switch (arraySchema.elementType.type) {
+        case SchemaType.Boolean:
+          return '[]bool';
+        case SchemaType.Date:
+          return '[]time.Time'
+        case SchemaType.DateTime:
+          return '[]time.Time';
+        case SchemaType.Duration:
+          return '[]time.Duration';
+        case SchemaType.Integer:
+          if ((<NumberSchema>schema).precision === 32) {
+            return '[]int32';
+          }
+          return '[]int64';
+        case SchemaType.Number:
+          if ((<NumberSchema>schema).precision === 32) {
+            return '[]float32';
+          }
```

Comment:
These cases might not be very common, but it would help in the future if an array of these types exists in a swagger